src/stp/new_plays/DefendPass.cpp

```diff
@@ -0,0 +1,149 @@
+//
+// Created by jordi on 14-05-20.
+//
+
+#include "stp/new_plays/DefendPass.h"
+
+#include "stp/invariants/BallCloseToThemInvariant.h"
+#include "stp/invariants/game_states/NormalPlayGameStateInvariant.h"
+#include "stp/new_roles/Defender.h"
+#include "stp/new_roles/Harasser.h"
+#include "stp/new_roles/Formation.h"
+#include "stp/new_roles/Keeper.h"
+
+namespace rtt::ai::stp::play {
+
+DefendPass::DefendPass() : Play() {
+    startPlayInvariants.clear();
+    startPlayInvariants.emplace_back(std::make_unique<invariant::NormalPlayGameStateInvariant>());
+    startPlayInvariants.emplace_back(std::make_unique<invariant::BallCloseToThemInvariant>());
+
+    keepPlayInvariants.clear();
+    keepPlayInvariants.emplace_back(std::make_unique<invariant::NormalPlayGameStateInvariant>());
```

Comment:
I think this means that we will only switch plays when we are not in normal_play

Reply:
Yes, you are right, the same for DefendShot. We somehow need a way to determine when we are done defending hmmm...

Reply:
Unrelated but damn we should figure something out for these disgusting emplace_back's with make_unique all over the place, it just looks _really_ bad.